dysnix/bsc/values.yaml

```diff
@@ -0,0 +1,140 @@
+# Default values for bsc.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+terminationGracePeriodSeconds: 180
+
+image:
+  repository: dysnix/bsc
+  tag: latest
+  pullPolicy: Always
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+# enable this only once - when your blockchain is empty. It overwrites everything
+init_genesis: true
+
+# Don't use spaces or special chars
+bsc:
+  # we don't have other chains, it's just a pod label for now
+  chain: testnet
+  #  Blockchain sync mode ("fast", "full", or "light") (default: fast)
+  syncmode: "full"
+  # Blockchain garbage collection mode ("full", "archive") (default: "full")
+  gcmode: "full"
+  base_path: "/data"
+  rpcApi: "eth,net,web3,txpool,parlia"
+  rpcVhosts: "*"
+  wsApi: "net,web3,eth"
+  wsOrigins: "*"
+  graphQlVhosts: "*"
+  maxpeers: 50
+  cache: 8192
+
+livenessProbe:
+  initialDelaySeconds: 300
+  periodSeconds: 300
+  timeoutSeconds: 10
+  successThreshold: 1
+  failureThreshold: 2
+
+# readinessProbe:
```

Comment:
why did we commented this section?

Reply:
During the CI we're running following command:

- `helm install bsc-g78bnxzb8x dysnix/bsc --namespace bsc-g78bnxzb8x --wait`

According to the documentation helm will wait for all pods been ready and healthy.
```txt
--wait - if set, will wait until all Pods, PVCs, Services, and minimum number of Pods of a Deployment, StatefulSet, or ReplicaSet are in a ready state before marking the release as successful. It will wait for as long as --timeout
```

It means that our helm install will run until it timeout'd if we set readiness probes, that's why it's commented in default `values.yaml` file, while it definitely should be set during production usage. Here is an example of [successful](https://github.com/dysnix/charts/runs/2857927708?check_suite_focus=true#step:5:75) run (when readiness probe is commented out) and [partially successful](https://github.com/dysnix/charts/runs/2857981125?check_suite_focus=true) but longer one (when readiness probe is enabled by default).

Please share your thoughts on this. Thanks in advance.

Reply:
It's ridiculous to get blockchain synced in CI :)
At the same time I think that our case "remove a lagging node from service" is rare, thus let's leave readiness disabled by default. Not to make CI happy, but to provide access to the node when it's not synced.